### PR TITLE
Improve code coverage and update PPP

### PR DESCRIPTION
**Pull request by LJXSean**
Added tests for RecordList and AddressBookParser which were previously lacking, in order to improve code coverage.

Updated PPP.

### CODE REVIEW

File: src/main/java/seedu/address/model/record/RecordList.java [hunk 1]
```diff
@@ -93,7 +96,15 @@ public boolean equals(Object other) {
                 && recordList.equals(((RecordList) other).recordList)); // state check
     }
 
+    /**
+     * Adds a Record to the list.
+     * The record must not already exist in the list.
+     */
```
Comment (EiffelLKF) [inline src/main/java/seedu/address/model/record/RecordList.java:102]: This javadocs might be wrong, should be delete record right?

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 2]
```diff
@@ -19,30 +19,21 @@
...
-            .build();
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline").withBirthdate("04-04-1944")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
             .withPhone("94351253")
             .withTags("friends").withAppointment("01-01-2024 0000").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier").withBirthdate("08-08-1988")
             .withAddress("311, Clementi Ave 2, #02-25")
             .withEmail("johnd@example.com").withPhone("98765432")
-            .withTags("owesMoney", "friends").withRecordList(RECORD1, RECORD2, RECORD3)
+            .withTags("owesMoney", "friends")
```
Comment (EiffelLKF) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:35]: Can we change the tags to include tags more related to OmniHealth?

File: src/main/java/seedu/address/model/record/RecordList.java (hunk 1 above)
Comment (LJXSean) [reply src/main/java/seedu/address/model/record/RecordList.java]: Yup, good catch!

**Review by EiffelLKF — APPROVED** [review]
LGTM

**Pull request merged**